package.json

```diff
@@ -89,6 +89,7 @@
     "proxyquire": "2.1.3",
     "semantic-release": "17.0.4",
     "sinon": "9.0.2",
+    "unist-util-map": "2.0.1",
```

Comment:
Where's that coming from?

Reply:
ha! (I thought you'll say something :-)

one of the `html.pre.js` has the `unist-util-map` as import, but since the test projects don't have a `package.json` and are not `npm installed`, it can't be found.

although it's a `helix-pipeline` dependency:
https://github.com/adobe/helix-pipeline/blob/4e279404aa665b72cb169b2be42baac57e9ac2bf/package.json#L84
it can't be imported. I guess this used to work previously, because `helix-pipeline` was a direct dependency of helix-cli.

also, I changed the behaviour of `package.cmd.js` to fail, if webpack has errors. this was _somehow_ disabled before. so many tests succeeded despite errors.

I'll create an issue to investigate why the dep is not seen by webpack.